test-suite/djinni/duration.yaml

```diff
@@ -26,6 +26,8 @@ java:
   reference: true
   generic: false
   hash: '%s.hashCode()'
+  writeToParcel: 'out.writeString(%s.toString())'
+  readFromParcel: '%s.parse(in.readString())'
```

Comment:
Same as with `Date` above. I think a better alternative is using `toNanos()` and `ofNanos(long)` instead of parsing strings.